testground/Dockerfile

```diff
@@ -0,0 +1,45 @@
+#:::
+#::: BUILD CONTAINER
+#:::
+
+# GO_VERSION is the golang version this image will be built against.
+ARG GO_VERSION=1.16
+
+# Dynamically select the golang version.
+FROM golang:${GO_VERSION}-buster
+
+# Testground version
+COPY VERSION /VERSION
+
+RUN apt-get update && apt-get install unzip
+
+WORKDIR /
+
+RUN wget "https://github.com/testground/testground/archive/refs/tags/v$(cat /VERSION).zip" && unzip "v$(cat /VERSION).zip"
+
+RUN  mv "testground-$(cat /VERSION)" app
+
+WORKDIR /app
+
+# Download deps.
+RUN go mod download
+
+COPY .env.toml /app/
+
+RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -ldflags "-X github.com/testground/testground/pkg/version.GitCommit=$(cat /VERSION)" -o testground
+
+#:::
+#::: RUNTIME CONTAINER
+#:::
+
+FROM golang:${GO_VERSION}-buster
+
+WORKDIR /app
+
+COPY --from=0 /app/testground /app/
+COPY --from=0 /app/.env.toml /root/testground/.env.toml
+ENV PATH="/app:${PATH}"
+
+EXPOSE 8042
+
+ENTRYPOINT [ "testground", "--vv", "daemon" ]
```

Comment:
I'm not sure we should have daemon in the entrypoint, testground should be enough

Reply:
But it's Dockerfile for testground daemon, we never use it for any other purpose plus it's how it was created by testground team